example/centos/centos-9-aarch64-image-installer.yaml

```diff
@@ -0,0 +1,576 @@
+otk.version: "1"
+
+otk.define:
+  # XXX these sizes might need to come from an external?
+  size_rootfs: '5368709120'
+  size_efiboot: '20971520'
```

Comment:
Maybe a comment like `# hardcoded in images` or something and a link to https://github.com/osbuild/images/blob/v0.89.0/pkg/image/anaconda_tar_installer.go#L24 here ?